pkg/aws/eni/routing/routing.go

```diff
@@ -42,6 +43,12 @@ var (
 // mtu: The ENI device MTU.
 // masq: Whether masquerading is enabled.
 func (info *RoutingInfo) Configure(ip net.IP, mtu int, masq bool) error {
+	if ip.To4() == nil {
+		log.WithFields(logrus.Fields{
+			"endpointIP": ip,
+		}).Warning("Unable to configure rules and routes because IP is not an IPv4 address")
+		return errors.New("Unable to configure rules, IP not compatible")
```

Comment:
Nit: let's keep error strings not capitalized and keep the reason short, e.g. `IP not compatible`. This lets the caller of the function fill in the context, like in this case, `configuring rules`.